content/tracing/product_specs/distributed_tracing.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Getting started: Distributed Tracing "
+kind: documentation
+---
+
+If you have read the [first example of tracing](/tracing) and you want to understand deeper how tracing works, let's take the following example which represents a simple API **thinker-api** and a micro-service behind it **thinker-microservice**. When the API receives a request with the correct *subject* parameter, it responds with a *thought*, otherwise, it responds with an error:
+
+{{< img src="tracing/miscellaneous/getting_started_distributed/tracing_overview_GS.jpg" alt="Tracing getting started overview" responsive="true" popup="true">}}
```

Comment:
Can we draw a diagram instead of having a picture of a whiteboard?

Reply:
Yup, I'm on it